test/Parse/line-directive.swift

```diff
@@ -53,3 +54,9 @@ enum E {
     case C, D
 #sourceLocation()
 }
+
+#sourceLocation(file: "sr8772.swift", line: 400)
+2., 3
+// CHECK: sr8772.swift:400:2: error: expected member name following '.'
```

Comment:
You can improve these checks using the [`[[@LINE]]` substitution](
https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-expressions), which allows relative line references in a CHECK pattern. In this case it'd be `[[@LINE-1]]`, `2`, `3`, etc.

Reply:
Would that work with #sourceLocation?

Reply:
`[[@LINE]]` is part of FileCheck, which doesn't know anything about Swift source. So it should work for this use case.

Reply:
…I'm sorry, I was giving feedback on autopilot. You're both quite right.